src/casp/services/bq_ops/extract/main.py

```diff
@@ -28,8 +28,11 @@ def extract_task(
     :param file_name: Name for a local file (.h5ad anndata) to save the output
     :param output_bucket_name: Name of GCS bucket
     :param output_bucket_directory: Directory in the bucket to save outputs
-    :param obs_columns_to_include:  Columns to include in `obs` data frame in extracted `anndata.AnnData` file
-        Mapped from extract query output
+    :param obs_columns_to_include: Optional list of columns from `cas_cell_info` table to include in ``adata.obs``.
+        If not provided, no specific columns would be added to ``adata.obs`` apart from `cas_cell_index`.
+        Note: It is required to provide the column names along with the aliases for the tables to which they belong.
+        However, the output extract table would contain only the column names, without any aliases.
+        Example: ``["c.cell_type", "c.donor_id", "c.sex", "i.dataset_id"]``
```

Comment:
Unrelated to this PR, but we should never simply catch a broad exception, print the error message, and move on. Let's fix this during the great big refactor.

Reply:
True